src/components/select/select.ts

```diff
@@ -0,0 +1,86 @@
+import { OnInit } from "@angular/core";
+import { BaseComponent, BaseElement, BaseOptions } from '../base';
+import { FormioComponents } from '../components';
+import { FormioTemplate } from '../../formio.template';
+import { CORE_DIRECTIVES, FORM_DIRECTIVES, NgClass } from '@angular/common';
+import { SELECT_DIRECTIVES } from '../../../node_modules/ng2-select/ng2-select';
+var Formio = require('formiojs');
+import 'style!ng2-select/components/css/ng2-select.css'
+
+export interface SelectOptions extends BaseOptions<any> {
+    placeholder?: string;
+    data: {};
+    dataSrc: string;
+    valueProperty?: string;
+    template: string;
+    refreshOn?: string;
+    filter?: any;
+    authenticate?: boolean;
+    searchField?: string;
+}
+
+export class SelectComponent extends BaseComponent<SelectOptions> {
+
+}
+
+interface IdTextPair{
+    id: string;
+    text: string;
+}
+
+let selectItems: IdTextPair[] = [];
+
+
+export class SelectElement extends BaseElement<SelectComponent> implements OnInit{
+    private value:any = {};
+    public refreshValue(value:any):void {
+        this.value = value;
+    }
+    ngOnInit(){
+        let template: string = this.component.settings.template.split('.')[1].split(' ')[0];
+        let valueProperty: string = this.component.settings.valueProperty;
+        switch(this.component.settings.dataSrc){
+            case 'values':
+                this.component.settings.data.values.forEach((item: any) => {
+                    selectItems.push({id: item.value, text: item.label});
+                });
+                this.component.settings.data.values = selectItems.slice(0);
+                break;
+            case 'json':
+                this.component.settings.data.json.forEach((item: any) => {
+                    selectItems.push({id: item[valueProperty], text: item[template]});
+                });
+                this.component.settings.data.values = selectItems.slice(0);
+                break;
+            case 'resource':
+                let baseUrl: string = 'https://ycxhcpsbzoefppz.form.io/';
```

Comment:
This is something that should be configured within the Formio.js library that is being pulled in. The FormioService handles the import https://github.com/formio/ng2-formio/blob/master/src/formio.service.ts. I recommend you will need to refactor this code so that it uses the Formio service and put a ```loadSubmissions``` method within this service. You can then configure the ```baseURL``` within the ```<formio>``` directive since you can pass in the service into that directive. Let me know if you need more direction than this.